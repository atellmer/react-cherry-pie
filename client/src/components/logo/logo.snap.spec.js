import React from 'react';
import renderer from 'react-test-renderer';

import TmLogo from './index';


jest.mock('react-dom');

test('Jest: TmLogo (Snapshot)', () => {
  const component = renderer.create(<TmLogo/>);
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
