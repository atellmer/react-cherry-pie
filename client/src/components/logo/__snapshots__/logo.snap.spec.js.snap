exports[`test Jest: TmLogo (Snapshot) 1`] = `
<div
  className={undefined} />
`;
